### PR TITLE
Create 2022-11-07-do-you-speak-prestashop-october-2022.md

### PR DESCRIPTION
**Pull request by l-delin**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

**Review by matks — APPROVED** [review]
@l-delin when should it be published?

**Comment by l-delin** [comment]
@matks not yet, I'm waiting for @Julievrz 's approval :-)

File: news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md
Comment (Julievrz) [inline news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md]: Careful with the "we": "we are looking for", "if you want to help us"... 
"We" could be understood as "we, the company". And the contributors are not working for the company. Be as specific as possible and rephrase the sentence to avoid using "we". Example: "Your project needs you", "Help the community", etc.

File: news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md
Comment (Julievrz) [inline news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md]: Same here :)

File: news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md
Comment (Julievrz) [inline news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md]: You could add a link to the Slack channel here.

File: news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md
Comment (Julievrz) [inline news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md:48]: The project news section is full... that's great! 🥳

File: news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md
Comment (Julievrz) [reply news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md]: You could also explain why proofreaders are needed. Almost fully translated but not validated, so, not available in the back office. Something like that. :)

**Comment by Julievrz** [comment]
> @matks not yet, I'm waiting for @Julievrz 's approval :-)

Sorry for the delay. I finished my review. It would be great to publish it tomorrow if you have time to read my comments because nobody will be working on Friday. 👀 

Thank you!

File: news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md
Comment (Julievrz) [inline news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md]:
```suggestion
date: 2022-11-10
```

File: news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md
Comment (Julievrz) [inline news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md]:
```suggestion
Your project needs you! Are you able to proofread for the Spanish (Argentina), Galician or Flemish projects? If so your help is needed: the projects are currently almost fully translated, but not approved. Reach me on either Crowdin or Slack, I’d be more than happy to guide you. 🙌
```

**Review by Julievrz — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Pull request merged**
